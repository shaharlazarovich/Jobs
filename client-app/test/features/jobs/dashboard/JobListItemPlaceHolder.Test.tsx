import { ReactWrapper } from "enzyme";
import { setupWrapper } from '../../../common/wrapper';
import React from "react";
import { findByTestAttr } from '../../../common/testUtils';
import JobListItemPlaceHolder from "../../../../src/features/jobs/dashboard/JobListItemPlaceHolder";

describe('Job List Item Place Holder UniTest', () => {
    let wrapper: ReactWrapper;
    beforeEach(async () => {
        wrapper = await setupWrapper(<JobListItemPlaceHolder  />);
    });
    test('renders without error', () => {
        const component = findByTestAttr(wrapper, 'component-job-list-item-place-holder');
        expect(component.length).toBe(1);
    });
    test('should match the snapshot', () => {
        const component = findByTestAttr(wrapper, 'component-job-list-item-place-holder');
        expect(component).toMatchSnapshot();
    });
})