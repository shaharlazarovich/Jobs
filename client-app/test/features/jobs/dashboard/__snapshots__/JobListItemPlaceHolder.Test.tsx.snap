// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Job List Item Place Holder UniTest should match the snapshot 1`] = `null`;
